components/airtable/new-records-in-view.js

```diff
@@ -0,0 +1,63 @@
+const airtable = require('https://github.com/PipedreamHQ/pipedream/components/airtable/airtable.app.js')
+const moment = require('moment')
+const axios = require('axios')
+const _ = require('lodash')
+
+module.exports = {
+  name: "new-records-in-view",
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    airtable,
+    baseId: { type: "$.airtable.baseId", appProp: "airtable" },
+    tableId: { type: "$.airtable.tableId", baseIdProp: "baseId" },
+    viewId: { type: "$.airtable.viewId", tableIdProp: "tableId" },
+  },
+  async run(event) {
+      const config = {
+        url: `https://api.airtable.com/v0/${this.baseId}/${encodeURIComponent(this.tableId)}`,
+        params: {
+          view: this.viewId,
+        },
+        headers: {
+          Authorization: `Bearer ${this.airtable.$auth.api_key}`,
+        },
+      }
+
+      let maxTimestamp = ""
+      const lastMaxTimestamp = this.db.get("lastMaxTimestamp") || ""
+      if (lastMaxTimestamp !== "") {
+        _.set(config,"params.filterByFormula", `CREATED_TIME() > "${lastMaxTimestamp}"`)
+        maxTimestamp = lastMaxTimestamp
+      }
+
+      const response = (await axios(config)).data
+
+      if (response.records.length > 0) {
```

Comment:
Losing an indent level makes this easier to read imo --

```
if (!response.records.length) {
  console.log(`No new records.`)
  return
}

// rest of the code with one less indent level
```

Reply:
done